TombLib/TombLib/LevelData/Compilers/TombEngine/Textures.cs

```diff
@@ -219,17 +272,18 @@ void WriteTextureData(BinaryWriterEx writer)
             var sky = GetSkyTexture();
             using (var ms = new MemoryStream())
             {
-                using (var bmp = sky.ToBitmap())
-                    bmp.Save(ms, System.Drawing.Imaging.ImageFormat.Png);
                 writer.Write(sky.Width);
                 writer.Write(sky.Height);
-                var output = RemoveColorChunks(ms);
```

Comment:
What was the purpose of `RemoveColorChunks` and why it was removed?

Reply:
My mistake :-( it caused also the dark textures bug.